src/System.Runtime.Extensions/tests/System/MathF.netcoreapp1.1.cs

```diff
@@ -26,6 +26,8 @@ public static partial class MathFTests
        // use CrossPlatformMachineEpsilon * 10.
         public const float CrossPlatformMachineEpsilon = 4.76837158e-07f;
 
+        private static unsafe bool Is64Bit => sizeof(void*) == 8;
```

Comment:
Can you just use `IntPtr.Size`? You wouldn't need to modify the project file below if you did that, as well.